src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -9,13 +9,12 @@ internal static class NumberFormatInfoData
         public static int[] UrINNumberGroupSizes()
         {
             if (
-                ((PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
+                (PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
 #if !uap
                 ||
                 (PlatformDetection.IsOSX && PlatformDetection.OSXKernelVersion >= new Version(15, 0))
 #endif
-                ) &&
-                (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610)
+                || PlatformDetection.IsUbuntu1510 || PlatformDetection.IsUbuntu1604 || PlatformDetection.IsUbuntu1610 || PlatformDetection.IsFedora
```

Comment:
(So i have no idea whether this is right for 1510 .. but I guess no tests will run so it doesn't matter.)